repo/packages/J/jenkins/0/marathon.json

```diff
@@ -0,0 +1,42 @@
+{
+  "id": "{{jenkins.framework_name}}",
+  "cpus": {{jenkins.cpus}},
+  "mem": {{jenkins.mem}},
+  "instances": 1,
+  "env": {
+      "CATALINA_OPTS": "{{jenkins.catalina_opts}}",
+      "JENKINS_FRAMEWORK_NAME": "{{jenkins.framework_name}}",
+      "JENKINS_CONTEXT": "/service/{{jenkins.framework_name}}",
+      "LD_LIBRARY_PATH": "/opt/mesosphere/lib"
+  },
+   "container": {
+       "type": "DOCKER",
+       "docker": {
+           "image": "mesosphere/jenkins:0.1.0",
```

Comment:
I think we want to bump this to `mesosphere/jenkins:0.1.3`, to match `currentVersion`